packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -54,6 +54,8 @@
 
   private boolean isInitialized = false;
 
+  private boolean isBuffering = false;
```

Comment:
I think `isBuffering` should be a state of `EventListener` instead of `VideoPlayer`. Let's move this ivar to the `EventListener`.

Reply:
Makes sense, have moved to inside `EventListener`, thanks